howto/How_To_Use_OpenOCD_UbuntuEd

```diff
@@ -0,0 +1,67 @@
+---
+title: Openocd On Ubuntu 64
+redirect_from:
+- /Openocd.html
+---
+This How to covers, if I rip my usb how do I flash the Proffieboard or update it. there's a few things about this particular use case. In this article you will be using the ST-Linkv2 usb debugger that connects through single core and the operating system of your laptop or pc would be Ubuntu 64 bit or any linux distro that is 64 bit. 
+
+on a windows box these steps are not required because of the openocd files are located in the arduino ide folders. the main issue with Ubuntu 64 bit version is that the scripts on arduino require the 32 bit version and ubuntu has deprecated libudev.so.1, THERE IS A FIX THOUGH.
+
+```
+./openocd.bin: error while loading shared libraries: libudev.so.1: cannot open shared object file: No such file or directory
+```
+
+pre-requisites
+the hardware required is the ST-Linkv2 debugger USB, plus the male/female pinned cables and small pcb test points connect the female pins into the usb pins that correspond with 1. sdio 2. swdlck and 3 gnd. the male end will go to the corresponding pins on the proffieboard. use the pcb test points to hold the pins in place other wise the connection may be lost during the compiling stage. 
```

Comment:
We could really really use an illustration or image on how this should be wired up.